Library/Homebrew/cmd/cleanup.rb

```diff
@@ -62,6 +62,10 @@ def cleanup_cache
         cleanup_path(path) { path.unlink }
         next
       end
+      if path.to_s.include?("java_cache")
```

Comment:
The code directly below that this mimics was added and reviewed very recently :wink:. Honestly, if someone can find me an example in the wild of a formula we download that happens to use `java_cache` in the file or folder name somewhere I'll send them the standard offering of :cake:. I'm perfectly happy to make the cache name a whole lot more random if desired.